<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_calc</class>
 <widget class="QDialog" name="credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>483</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>483</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>196</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Общая сумма кредита:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>184</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ежемесяный платеж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>170</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>13</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>48</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Срок:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>197</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Переплата по кредиту:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
